### ISSUE TITLE
Les attributs grades et averages renvoient 0

### ISSUE THREAD

**Issue by ChristopheHELLEQUIN** [delphiki/hass-pronote#92]
Bonjour,

Depuis récemment, les attributs grades et averages des entités générées par l'intégration renvoient systématiquement 0, alors qu'ils devraient contenir les données de notes et moyennes.

Informations :

Versions de Home Assistant : 
Core
2024.11.3
Supervisor
2024.11.4
Operating System
13.2
Interface utilisateur
20241106.2

Version de l'intégration Pronote : 0.15.0

Environnement : Home Assistant OS sur Freebox Delta VM

Je souhaiterais savoir si d'autres utilisateurs rencontrent le même problème et s'il y a des pistes de résolution.

Merci d'avance pour votre aide !

**Comment by delphiki** [comment]
Hello,

Je pense que c'est dû au changement de trimestre :)
Il faudrait effectivement trouver une solution pour rapatrier les notes des trimestres finis.

**Comment by ChristopheHELLEQUIN** [comment]
Je vais patienter, même si je n'avais pas rencontré ce problème lors des précédents changements de trimestre.
Merci pour votre aide et pour tout le travail accompli sur cette intégration !

**Comment by NuoLeonis** [comment]
Effectivement, les notes ne sont toujours pas dispos pour mon lycéen, pour le collégien en semestres elles sont là
